src/screens/SearchScreen.js

```diff
@@ -9,19 +9,21 @@ import {initializeMap, updateViewStyle, mapLoaded, updateRegion} from "../action
 
 import _ from 'lodash';
 
-import {CardSection, Card} from '../components/common/index';
```

Comment:
Trying to get rid of all of our hand-made components. Every screen I refactor I'll be removing them until we can finally just rid of the entire folder.

Reply:
yea, that's a bit annoying, it's on my list to replace as many of the old components as possible today and tomorrow.